static/app/components/replays/replaySession.tsx

```diff
@@ -0,0 +1,86 @@
+import invariant from 'invariant';
+
+import {ProjectAvatar} from 'sentry/components/core/avatar/projectAvatar';
+import {UserAvatar} from 'sentry/components/core/avatar/userAvatar';
+import {Grid} from 'sentry/components/core/layout';
+import {Flex} from 'sentry/components/core/layout/flex';
+import {Text} from 'sentry/components/core/text';
+import TimeSince from 'sentry/components/timeSince';
+import {IconCalendar} from 'sentry/icons/iconCalendar';
+import {IconDelete} from 'sentry/icons/iconDelete';
+import {t} from 'sentry/locale';
+import * as events from 'sentry/utils/events';
+import useProjectFromId from 'sentry/utils/useProjectFromId';
+import type {ReplayListRecordWithTx} from 'sentry/views/performance/transactionSummary/transactionReplays/useReplaysWithTxData';
+import type {ReplayListRecord} from 'sentry/views/replays/types';
+
+interface Props {
+  replay: ReplayListRecord | ReplayListRecordWithTx;
+}
+
+export default function ReplaySession({replay}: Props) {
```

Comment:
Session Replay Session

Reply:
needs a better name :(

Reply:
like Replay Badge?

kinda chunky compared to other badges.

Reply:
yeah that badge makes sense, nbd either way

Reply:
`replayBadge.tsx` for now!